packages/nuxt-locokit/playground/pages/404.vue

```diff
@@ -1,6 +1,13 @@
 <template>
-  <div>
-    404 Page
-    <p>{{ $route.params.slug }}</p>
-  </div>
+  <WithBackground>
+    <div class="p-6 bg-white rounded-lg border border-gray-200 shadow-md dark:bg-gray-800 dark:border-gray-700 text-center">
+      <h1 class="mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white">{{ $t('pages.404.title') }}</h1>
+      <p class="mb-7 font-normal text-gray-700 dark:text-gray-400">{{ $route.params.slug }}</p>
```

Comment:
In nuxt3, we need to use `useRoute()` to retrieve params from route. 
I don't know if $route is still maintained/depreciated. :thinking:

Reply:
we have removed it because it's useless